packages/near-membrane-base/src/membrane.ts

```diff
@@ -611,6 +644,44 @@ export function createMembraneMarshall() {
             return fn;
         }
 
+        function isLiveObject(value: any): boolean {
+            if (value === null || value === undefined) {
+                return false;
+            }
+            if (typeof value === 'object') {
+                if (
+                    // Arrays are marked as live in the BoundaryProxyHandler constructor.
+                    // isArrayOrNotOrThrowForRevoked(value) ||
+                    ArrayBufferIsView(value) ||
+                    isPlainObject(value)
+                ) {
+                    return true;
+                }
+            }
+            if (ReflectApply(ObjectProtoHasOwnProperty, value, [LOCKER_LIVE_VALUE_MARKER_SYMBOL])) {
```

Comment:
this should be the less common case, probably should be last on this method.

Reply:
Done.